src/app/AttributeCache.h

```diff
@@ -360,10 +360,13 @@ class AttributeCache : protected ReadClient::Callback
         return mCallback.OnDeallocatePaths(std::move(aReadPrepareParams));
     }
 
+    uint32_t OnUpdateDataVersionFilterList(DataVersionFilterIBs::Builder & aDataVersionFilterIBsBuilder,
+                                           DataVersionFilter * apDataVersionFilterList, size_t aDataVersionFilterListSize) override;
+
 private:
     Callback & mCallback;
     NodeState mCache;
-    std::set<ConcreteAttributePath> mChangedAttributeSet;
+    std::set<ConcreteDataAttributePath> mChangedAttributeSet;
```

Comment:
Does list chunking have a data version in every chunk?

Reply:
data version is put inside attributeDataIB, yes, every chunk, it has data version for each attribute data IB